src/screens/Sablier/IncomingStream.js

```diff
@@ -0,0 +1,199 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
```

Comment:
Is there any difference in performance between `import * as React from 'react'` and `import React from 'react'`? I think in both cases we're just importing React. And React must be imported if we use JSX syntax, if not we'd get an error. What do you mean by optimisation here?

Reply:
Actually I'm not sure what else is imported where when you use `*` - but I was thinking that it would be more than just `React` into the module. My thinking here was to decrease the memory footprint by only including the modules needed from `react` as opposed to _everything_..